actionmailer/lib/action_mailer/log_subscriber.rb

```diff
@@ -6,9 +6,16 @@ module ActionMailer
   class LogSubscriber < ActiveSupport::LogSubscriber
     # An email was delivered.
     def deliver(event)
-      info do
-        recipients = Array(event.payload[:to]).join(', ')
-        "\nSent mail to #{recipients} (#{event.duration.round(1)}ms)"
+      recipients = Array(event.payload[:to]).join(', ')
+
+      if exception = event.payload[:exception]
+        error do
+          "\n#{exception.first} was raised when sending mail to #{recipients} (#{duration_in_ms(event)})"
```

Comment:
is `exception` always guaranteed to be an array? If not, you can wrap it in `Array()` similar to `recipients`.